tests/integration/test_orders_cli.py

```diff
@@ -320,12 +317,13 @@ def test_cli_orders_create_basic_success(
         '--id', id_string,
         '--bundle', 'analytic',
         '--item-type', 'PSOrthoTile'
-        ])
+        ])  # yapf: disable
```

Comment:
What do we do with cli options that are given as a list, where [OPTION NAME] and [OPTION VALUE] are separate entries? yapf wants to put them all on different lines. See what happens when it gets its way [here](https://github.com/planetlabs/planet-client-python/pull/405/files#r819824573). I had decided to just go along with yapf but accidentally left this (and another, below) disable here. It's a good discussion point so I'll leave it for now, but my inclination would be to remove it.